packages/xchain-thorchain/src/util.ts

```diff
@@ -98,6 +98,8 @@ export const getPrefix = (network: Network) => {
   switch (network) {
     case Network.Mainnet:
       return 'thor'
+    case Network.Stagenet:
+      return 'sthor'
     case Network.Testnet:
       return 'tthor'
```

Comment:
Pls add a test for `getPrefix` (we don't have it yet)

Reply:
Test added in `util.test.ts`. Resolved.